modules/aws/vpc/vpc-subnet-worker.tf

```diff
@@ -24,6 +24,19 @@ resource "aws_subnet" "worker_1" {
   )}"
 }
 
+resource "aws_subnet" "worker_2" {
```

Comment:
These resources here looks pretty redundant. Would be possible to create an unique resource `aws_subnet` and leverage interpolation?

Reply:
Not sure if you can interpolate inside the resource name.

You can create modules or one resource with count.
Anyway it is always a good idea to reduce redundant code.

Reply:
> Not sure if you can interpolate inside the resource name.

No you can't. Terraform resource name should not know anything about the count of the resource. we should call it simply `worker` and then interpolate with `count` as you said

Reply:
Yep, good suggestion, I think we should be able to do that,  but we would need to move subnet values into array